modules/virt-deleting-deployment-custom-resource.adoc

```diff
@@ -2,37 +2,20 @@
 //
 // * virt/install/uninstalling-virt-web.adoc
 
-//This file contains UI elements and/or package names that need to be updated.
-
 :_content-type: PROCEDURE
 [id="virt-deleting-deployment-custom-resource_{context}"]
-= Deleting the {VirtProductName} Operator Deployment custom resource
-
-To uninstall {VirtProductName}, you must first delete the *{VirtProductName} Operator Deployment* custom resource.
-
-.Prerequisites
+= Deleting the HyperConverged custom resource
 
-* Create the *{VirtProductName} Operator Deployment* custom resource.
+To uninstall {VirtProductName}, you first delete the `HyperConverged` custom resource (CR).
 
 .Procedure
 
-. From the {product-title} web console, select `openshift-cnv` from the *Projects* list.
-
 . Navigate to the *Operators* -> *Installed Operators* page.
 
-. Click *{VirtProductName}*.
+. Click the {VirtProductName} Operator.
```

Comment:
I'm not sure I see the difference between click and select. The Operator appears as a "card" and you have to click it. I admit that I did not check ISG because Patternfly seems to have a lot more UI elements than IBM uses.

I don't mind changing it but the logic is not clear to me. :-)

Reply:
The way I see it the **Installed Operators** page has a list of Operators and the ISG recommends using _select_ to refer to selecting an item from a list. That was my logic :-)

Reply:
Really? I see squares or cards, not a list.